app/views/ui-components/v1/navs/_primary_nav.html.slim

```diff
@@ -92,3 +93,17 @@
               li.pl-1 role="presentation" class="#{pundit_class HbxProfile, :view_admin_tabs?}"
                 = link_to(notifier.notice_kinds_path, "aria-expanded" => "true", "aria-controls" => "notices", "role" => "tab" ) do
                   | Notices
+
+
+javascript:
+  document.addEventListener('DOMContentLoaded', function() {
+    window.localStorage.removeItem("jwt");
+    var agencyStaffLink = document.getElementById('agencyStaffLinkToExternal');
+    agencyStaffLink.addEventListener('click', function(event) {
```

Comment:
Do we have to wait for a click to set the localstorage jwt? Could we set it as soon as the DOM ContentLoaded is fired?